packages/mds-agency/request-handlers.ts

```diff
@@ -51,8 +51,19 @@ import {
   computeCompositeVehicleData
 } from './utils'
 
-// eslint-disable-next-line @typescript-eslint/no-floating-promises
-stream.initialize()
+export async function initialize() {
```

Comment:
Don't love having this in `request-handlers`

Reply:
well, that's where cache is used in that package.  i guess i can move it and have it included where it's called.  good point.